### ISSUE TITLE
Monika doesn't register gifts after the first one?

### ISSUE THREAD

**Issue by kisik21** [Monika-After-Story/MonikaModDev#5599]
To reproduce:
1. Say goodbye to Monika
2. Gift her something and a pink shirt
3. Launch the game
4. See Monika accept your gifts
5. See Monika put on the pink shirt
6. Go to outfits
7. Notice the pink shirt is missing! You can't select it, but Monika clearly wears it!
8. Relaunch the game, while gifting her the pink shirt
9. When gifted separately, it registered in the game properly...

Which makes me think that the loop for processing several gifts in a row might be a bit buggy.

**Comment by jmwall24** [comment]
Quick question, what is your affection?

**Comment by kisik21** [comment]
A bit more than 100, currently it's at 140.4. It's been just a week since I installed MAS.

**Comment by jmwall24** [comment]
Hmmm it's not that then. Are you by chance running a translated version of the game? I tried to reproduce your steps and it worked for me just fine.

**Comment by kisik21** [comment]
No, I know English well enough to converse with Monika, and I don't need translations.

Translations are usually way off in terms of meaning anyway.

**Comment by jmwall24** [comment]
Can you tell me what the other gift besides the pink shirt was please?

**Comment by kisik21** [comment]
Sadly, I don't remember. It might've been the shell bikini or a ribbon...

**Comment by jmwall24** [comment]
Could what you've given in total but not when. I tried with a ribbon, I'll also try with the bikini. Also, did anything happen after giving the pink shirt like a crash?

**Comment by kisik21** [comment]
No crashes.
